.travis.yml

```diff
@@ -29,14 +29,17 @@ jobs:
     - stage: test
       script:
         # decrypt archive containing the kakadu library
-        - openssl aes-256-cbc -K $encrypted_53935862068b_key -iv $encrypted_53935862068b_iv -in vendor/v7_A_2-01382N.zip.enc -out vendor/v7_A_2-01382N.zip -d
+        - openssl aes-256-cbc -K $encrypted_53935862068b_key -iv $encrypted_53935862068b_iv -in vendor/v7_A_3-01382N.zip.enc -out vendor/v7_A_3-01382N.zip -d
```

Comment:
@subotic I think this should be `v7_A_4-01727L.zip.enc`, right?

Reply:
https://github.com/dhlab-basel/Sipi/pull/229/commits/1551367c56312ee7012d8806e1ea69a7e595a31b thanks!

Reply:
your welcome :-)